ballerina-test-automation/installer-test/src/test/java/io/ballerina/installer/test/TestUtils.java

```diff
@@ -55,7 +55,7 @@ public static String getVersionOutput(String jBallerinaVersion, String specVersi
                                           String versionDisplayText) {
         String toolText = TestUtils.isOldToolVersion(toolVersion) ? "Ballerina tool" : "Update Tool";
         if (jBallerinaVersion.contains(TestUtils.SWAN_LAKE_KEYWORD)) {
-            return "Ballerina " + versionDisplayText + " (Swan Lake)\nLanguage specification "
+            return "Ballerina " + versionDisplayText + " (Swan Lake Update 1)\nLanguage specification "
```

Comment:
Shall we get this dynamically from the version itself. We can use the same logic used in the version command

https://github.com/ballerina-platform/ballerina-lang/pull/36262/files

Reply:
@keizer619 sure.

Reply:
@keizer619 addressed the suggestions.